js/apps/admin-ui/cypress/e2e/clients_test.spec.ts

```diff
@@ -231,6 +231,94 @@ describe("Clients test", () => {
     });
   });
 
+  describe("Client scopes evaluate subtab", () => {
+    const clientName = "testClient";
+
+    beforeEach(() => {
+      loginPage.logIn();
+      keycloakBefore();
+      commonPage.sidebar().goToClients();
+    });
+
+    before(() => {
+      adminClient.createClient({
```

Comment:
Although this likely won't cause issues, the Promise here should either be `await`ed or returned directly. The reason for this is that Cypress will be able to wait for the client to be created, before the tests start running.